ribbon-loadbalancer/build.gradle

```diff
@@ -4,13 +4,14 @@ dependencies {
     api "io.reactivex:rxjava:${rx_java_version}"
     api "org.slf4j:slf4j-api:${slf4j_version}"
     api "com.netflix.servo:servo-core:${servo_version}"
-    api "com.google.guava:guava:${guava_version}"
     api 'com.netflix.netflix-commons:netflix-commons-util:0.1.1'
+    api 'com.github.ben-manes.caffeine:caffeine:2.9.2'
```

Comment:
Not sure if we have to involve lawyers but this does change our license structure slightly.  The fact that it is used in Spring and Cassandra likely means it's probably permissible.